assets/scss/markmap.scss

```diff
@@ -0,0 +1,4 @@
+.markmap > svg {
+    width: 800px;
+    height: 500px;
```

Comment:
> I get the impression that it would be preferable to restrict the with of `div.markmap` via `@extend .td-max-width-on-larger-screens` when inside a `.td-content`, rather than bound the with of the svg as is done here.

Yes, goo idea. I inherited this code though, all I did was moving it out into separate, new file `assets/scss/markmap.scss`.
I would prefer to address your suggestion in a separate issue.